workshop/content/60-construct-hub/_index.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Construct Hub"
+weight = 60
+chapter = true
++++
+
+# The Construct Hub
+
+The [CDK Construct Hub](https://constructs.dev/) is a one-stop destination for finding, reusing and sharing constructs authored by AWS, [AWS Partner Network partners](https://aws.amazon.com/partners/), third parties, and the developer community. The site currently lists constructs for our supported programming languages Typescript, Java, Python, and .NET. CDK constructs are cloud architecture building blocks and patterns that you can use to stand up complete production-ready cloud applications. The constructs listed in the Construct Hub are authored using the [AWS Cloud Development Kit](https://aws.amazon.com/cdk/) (AWS CDK), [CDK for Kubernetes](https://cdk8s.io/) (CDK8s) and [CDK for Terraform](https://github.com/hashicorp/terraform-cdk) (CDKtf).
+
+We define [constructs](https://docs.aws.amazon.com/cdk/latest/guide/constructs.html) as classes, which define a “piece of system state”. Constructs can be composed together to form higher-level building blocks which represent more complex state. AWS, enterprises, start-ups, and individual developers use CDK constructs to share proven architecture patterns as reusable code libraries, so that everyone can benefit from the collective wisdom of the community.
+
+The Construct Hub is the central location where CDK users can find a comprehensive collection of constructs to help them build their applications. The Construct Hub makes it easier for developers to find the high-level building blocks they need to build their applications by listing publicly released construct libraries.
+
+## Discovering and Using Constructs
+Browse to https://constructs.dev and search for constructs based on keywords such as names of AWS services used (e.g., “eks”, “dynamodb”), the library’s author (e.g., “pahud”), or the target provisioning engine (e.g., “cdktf”, “cdk8s”). Examples of constructs you can find include datadog-cdk-constructs that instruments Python and Node.js Lambda functions with Datadog, cdk-gitlab-runner that creates a GitLab Runner and executes a pipeline job, cdk-k3s-cluster that deploys a K3s cluster, and many more. The Construct Hub website also includes links to Getting Started resources. Below is an example of search results for constructs that include the “bucket” keyword and support the Python programming language:
```

Comment:
Actually cdktf and cdk8s are the CDK types and not the provisioning engine (CloudFormation, Terraform, Kubernetes). Perhaps we should considering adding a specific filter for that